lldb/examples/python/crashlog.py

```diff
@@ -1720,6 +1735,13 @@ def CreateSymbolicateCrashLogOptions(
         help="show source for all threads, not just the crashed thread",
         default=False,
     )
+    arg_parser.add_argument(
+        "--no-parallel-image-loading",
+        dest="parallel_image_loading",
+        action="store_false",
+        help=argparse.SUPPRESS,
+        default=True,
```

Comment:
Does this mean parallel loading is on or off by default? The negation confuses me.